backend/src/main/java/hu/flowacademy/konyhatunder/service/RecipeService.java

```diff
@@ -51,25 +52,25 @@ public void save(EmptyRecipe emptyRecipe) {
                 .categoryList(categoryList)
                 .build());
     }
-    @SneakyThrows
-    public void validate(EmptyRecipe emptyRecipe){
-        if (!StringUtils.hasText(emptyRecipe.getName()))
-            throw new ValidationException("A recept nevét kötelező megadni");
 
-        if(!StringUtils.hasText(emptyRecipe.getDescription()))
-            throw new ValidationException("Ez elkészités mező nem lehet üres");
+    public List<String> getAllRecipeLevels() {
+        Level[] levels = Level.values();
+        return Arrays.stream(levels).map(Enum::name).collect(Collectors.toList());
+    }
 
-        if(emptyRecipe.getPreparationTime() <= 0)
-            throw new ValidationException("Elkészitése idő nem lehet 0 perc");
+    @SneakyThrows
+    public void validate(EmptyRecipe emptyRecipe) {
+        if (!StringUtils.hasText(emptyRecipe.getName()))
+            throw new ValidationException("A recept nevét kötelező megadni!");
 
-        if(emptyRecipe.getLevel() == null)
-            throw new ValidationException("Nem jó level");
+        if (!StringUtils.hasText(emptyRecipe.getDescription()))
+            throw new ValidationException("Ez elkészités mező nem lehet üres!");
 
-    }
+        if (emptyRecipe.getPreparationTime() <= 0)
+            throw new ValidationException("Elkészitése idő nem lehet 0 perc!");
```

Comment:
Pontatlan üzenet, "nem lehet 0-nál kisebb". Kicsit magyartalan is, "Elkészítési idő" a korrekt.

Reply:
Módosítva